test/commands/bg/bg.collection.create.test.ts

```
@@ -24,6 +29,31 @@ describe('bg collection create command', () => {
         expect(signature).to.match(/^[a-zA-Z0-9]{32,}$/)
     })
 
+    it('creates a collection with a vanity --mint-keypair', async () => {
+        const umi = createUmi(TEST_RPC)
+        const mintKeypair = generateSigner(umi)
+        const mintKeypairPath = join(os.tmpdir(), `mplx-test-mint-${Date.now()}.json`)
+        fs.writeFileSync(mintKeypairPath, JSON.stringify(Array.from(mintKeypair.secretKey)))
```

ESLint

[error] Prefer the spread operator over `Array.from(…)`.

(unicorn/prefer-spread)